doc/getting_started/installation_win.rst

```diff
@@ -168,11 +168,12 @@ packages from their respective websites.
 This should check that all the tools and toolchain are set up correctly for
 your own Zephyr development.
 
-Using MSYS2
+Option 2: MSYS2
```

Comment:
The title underline must be at least as long as the title, so add some more ``====`` 's

Reply:
Done